default.nix

```diff
@@ -0,0 +1,75 @@
+{ nixpkgsBranch ? "release-21.05"
+, nixpkgs ? "https://github.com/NixOS/nixpkgs/archive/refs/heads/${nixpkgsBranch}.tar.gz"
+, python ? "python39"
+, rfVersion ? "3.2.2"
+, path ? toString ./.
+, cmd ? "invoke test --in-nix" }:
+let
+  pkgs = import (fetchTarball nixpkgs) { };
+in
+with pkgs;
+with lib;
+let
+  mach-nix = import (builtins.fetchGit {
+    url = "https://github.com/DavHau/mach-nix/";
+    ref = "refs/tags/3.3.0";
+  }) {
+    inherit pkgs python;
+    pypiDataRev = "c8393888d97e74f2217aaafae70bf5bc5c913535";
+    pypiDataSha256 = "0pfivp1w3pdbsamnmmyy4rsnc4klqnmisjzcq0smc4pp908b6sc3";
+  };
+
+  requirementsWithout = names:
```

Comment:
Maybe explaining comment here? "Utility function to filter certain dependencies from the rest" or something similar